src/components/ModelCamera/components/Model.ts

```diff
@@ -0,0 +1,75 @@
+import * as tf from "@tensorflow/tfjs";
+import { Models } from "../../../types/Models";
+import { isMobile } from 'react-device-detect';
+import * as tflite from '@tensorflow/tfjs-tflite'
+// Adds the CPU backend.
+//import '@tensorflow/tfjs-backend-cpu';
+export async function loadModel() {
+    let result: Models = { R_Model: undefined, L_Model: undefined };
+
+    if (isMobile) {
+        tflite.setWasmPath(
+            'https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-tflite@0.0.1-alpha.8/dist/'
+        );
+        console.log('isMobile');
+        result.R_Model = await tflite?.loadTFLiteModel('https://tflite-asl-models.s3.amazonaws.com/R_converted_model.tflite')
+        result.L_Model = await tflite?.loadTFLiteModel('https://tflite-asl-models.s3.amazonaws.com/L_converted_model.tflite')
+    } else {
+        console.log("isDesktop");
+        result.R_Model = await tf?.loadGraphModel(
```

Comment:
Curious as to why tf has an optional chain, as it is imported from a library.

Reply:
not sure. Used old code for this. I can try to remove it to see if it works